pkg/cni/install.go

```diff
@@ -70,17 +70,22 @@ func NewInstaller(mode string,
 
 func (i *Installer) Start() error {
 	if i.Mode == constants.AdsMode || i.Mode == constants.WorkloadMode {
```

Comment:
Are there other modes?

Reply:
can not specify any mode, like if we want to enable bypass